optuna_dashboard/ts/components/AppDrawer.tsx

```diff
@@ -246,6 +248,24 @@ export const AppDrawer: FC<{
                 <ListItemText primary="Analytics" sx={styleListItemText} />
               </ListItemButton>
             </ListItem>
+            {studyDetail?.is_preferential && (
+              <ListItem key="PreferenceGraph" disablePadding sx={styleListItem}>
+                <ListItemButton
+                  component={Link}
+                  to={`${URL_PREFIX}/studies/${studyId}/graph`}
+                  sx={styleListItemButton}
+                  selected={page === "graph"}
+                >
+                  <ListItemIcon sx={styleListItemIcon}>
+                    <LanIcon />
+                  </ListItemIcon>
+                  <ListItemText
+                    primary="PreferenceGraph"
```

Comment:
How about this?

```suggestion
                    primary="Preferences (Graph)"
```

Reply:
I actually do not prefer "PreferenceHistory" and "HumanInTheLoop". At least there should be a space between each word. What do you think?